### DISCUSSION TITLE
Pulp Solver Error

### DISCUSSION THREAD

**Discussion by arunav71** [coin-or/pulp#600] [Q&A]
Hello everyone,

I was trying to solve an optimization using Pulp but ran into this Solver error

1. I am running this on Jupyter notebook
2. I tried uninstalling and reinstalling Pulp both on Windows and then removing the pulp package from Jupyter but it still didn't work
3. As you can see below, my problem gets set up correctly. But I get this error when I try to solve it

Any help/suggestions will be greatly appreciated.

Thanks,

![image](https://user-images.githubusercontent.com/36560507/200756393-fd3ac9f7-a6d0-4ff1-9ce2-3a75e27ccf41.png)
![image](https://user-images.githubusercontent.com/36560507/200756499-8c3d8708-9939-4786-be0c-78ad8992d28c.png)

**Comment by pchtsp** [comment]
Have you checked https://coin-or.github.io/pulp/guides/how_to_debug.html?

**Reply by arunav71** [reply]
Thanks for your response @pchtsp . And yes,I had looked at that link while trying to troubleshoot this. As you can see in the snip below, I had done both the things they suggested
1. Install Pulp per documentation [using "pip install pulp"]
2. Install pulp in the same folder as Jupyter [Users directory]

But I still keep getting this error.

Thanks,

![image](https://user-images.githubusercontent.com/36560507/200943771-596b9d2a-caf7-4f16-830f-a52639d76082.png)
